implementations/rust/ockam/ockam_command/src/run/parser/mod.rs

```diff
@@ -0,0 +1,339 @@
+mod nodes;
```

Comment:
Maybe this is only a personal preference but I'd rather see no "domain" code in a `mod` file, and instead have a `config.rs` file + a `resources.rs` file inside the `parser` folder.

Reply:
Thanks for the feedback 🙏  I'll leave this one for the follow-up PR because I'm making a lot of changes here. I'll address the other two comments in this PR.